cylc/flow/network/graphql.py

```diff
@@ -226,12 +225,22 @@ def args_selection_search(
 
 
 def execute_and_validate_and_strip(
-        schema,  # type: GraphQLSchema
-        document_ast,  # type: Document
-        *args,  # type: Any
-        **kwargs  # type: Any
```

Comment:
Flake8 expects these types to be imported? Simple solution move them to a Google-style docstring like we use elsewhere.